working-group-abilities.md

```diff
@@ -0,0 +1,44 @@
+# Working Group Abilities
+
+Established working groups in the OpenSSF are given broad latitude to accomplish their goals.
+The TAC aims to only require formal approval in limited cases.
+This document outlines some of the explicitly permitted activities and resources made
+available to working groups.
+It is not intended to be exhaustive, and will grow over time.
+
+When in doubt, reach out to the TAC with any questions!
+
+## Repositories
+
+We expect that working groups will need to create repositories for code, tools, and other projects.
+These should be created in the [ossf](github.com/ossf) organization where possible to simplify
+management.
+
+Working groups can also request new organizations where necessary - reach out to the TAC for
+logistics.
+
+
+If any of these projects grow large enough, they can be "spun-off" into separate working groups by
+the TAC.
+As a general rule-of-thumb, if a sub-project requires its own standing meeting, it might be time to
+spin-it-off.
+
+## Papers
+
+As the working groups are intended to be the subject matter experts in their domains,
+they should author papers for release through the OpenSSF with no formal review or approval.
+However, the TAC does ask that drafts be shared as early as possible for feedback and awareness.
```

Comment:
Should we provide more clarity on who or how the papers should be shared for feedback and awareness?  Or, do we want to wait until the communications/marketing committee is established to provide guidance?

Reply:
I can't think of anything more concrete to recommend, any suggestions?

Reply:
How about something like this "... the TAC does ask that drafts be shared as early as possible (ideally at least 7 days in advance of publication) for feedback and awareness by sending email to openssf-tac@lists.openssf.org. The TAC can also help arrange visibility the papers via public announcements, blog posts and tweets."

Reply:
I think I got this in.